src/summary/cards/vertical-label-value.component.tsx

```diff
@@ -4,11 +4,15 @@ import styles from "./vertical-label-value.css";
 export default function VerticalLabelValue(props: VerticalLabelValueProps) {
   return (
     <div className={styles.root}>
-      <label className={`omrs-type-body-small ${styles.label}`}>
+      <label
+        className={
+          props.labelClassName || `omrs-type-body-small ${styles.label}`
+        }
+      >
         {props.label}
       </label>
       <div
-        className={props.className}
+        className={props.valueClassName || props.className}
```

Comment:
I think we should choose one between `className` and `valueClassName`, having both might be confusing.

Reply:
Agreed 👍